<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="SettingsTab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>491</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="CameraSettingsTab">
    <attribute name="title">
     <string>Camera Settings</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="spPixelSize">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size of pixel in microns</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Fuducial Marks</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>top left</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>top right</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>bottom left</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fm_TL_X">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fm_TL_Y">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fm_TL">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Display</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fm_TR_Y">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fm_TR">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Display</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fm_TR_X">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fm_BL_Y">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fm_BL">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Display</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_fm_BL_X">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>150</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="StageSettingsTab">
    <attribute name="title">
     <string>Stage Settings</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="TargetSettingsTab">
    <attribute name="title">
     <string>Target Settings</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Algorithm Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_distance">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>158</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCluster">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>130</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QComboBox" name="AlgoComboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
     </property>
     <property name="text">
      <string>COOC (Colour Cooccurrance) Settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="ScoreThresholdLbl">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>185</y>
       <width>18</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>185</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSlider" name="numClusters">
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="distanceThreshold">
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="ScoreThreshold">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>70</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>59</width>
       <height>81</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>No. Clusters</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Distance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Score
Threshold</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="ThresholdSettingsTab">
    <attribute name="title">
     <string>Threshold Settings</string>
    </attribute>
    <widget class="QSlider" name="ThresholdMinSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>160</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;All values &lt;span style=&quot; font-weight:600;&quot;&gt;above&lt;/span&gt; this threshold are white in the image otherwise black&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ThresholdMinLbl">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold Min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Threshold Max</string>
     </property>
    </widget>
    <widget class="QSlider" name="ThresholdMaxSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>160</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;All values &lt;span style=&quot; font-weight:600;&quot;&gt;below&lt;/span&gt; this threshold are white in the image otherwise black&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>254</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ThresholdMaxLbl">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>255</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkAutoThreshold">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatic Thresholding</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="histogram_image">
     <property name="geometry">
      <rect>
       <x>276</x>
       <y>50</y>
       <width>171</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="lineThresholdMin">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>50</y>
       <width>20</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="lineThresholdMax">
     <property name="geometry">
      <rect>
       <x>437</x>
       <y>50</y>
       <width>20</width>
       <height>181</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="NoClustersSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>160</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The image will be posterized into the set number of gray levels. When No. levels = 2, this isthe same as performing a binary threshold.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of levels</string>
     </property>
    </widget>
    <widget class="QLabel" name="NoClustersLbl">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>230</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Connected Components</string>
    </attribute>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How much smaller can the detected targets be than the example target.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Size tolerance (% smaller)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How much bigger can the detected targets be than the example target.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Size tolerance (% larger)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sizeToleranceSmallerSpinBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How much smaller can the detected targets be than the example target.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sizeToleranceLargerSpinBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>65</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;How much bigger can the detected targets be than the example target.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hu moments are calculated for the binary shape, these are shape measures. The similarity setting determines how important similar shape of the detected targets are (100% =  shape must be the same, 0% = shape not important).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Hu Moment Similarity (% similar)</string>
     </property>
    </widget>
    <widget class="QSlider" name="huMomentSimilaritySlider">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>160</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hu moments are calculated for the binary shape, these are shape measures. The similarity setting determines how important similar shape of the detected targets are (100% = shape must be the same, 0% = shape not important).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="resetValues">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>362</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Values</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ThresholdMinSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ThresholdMinLbl</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ThresholdMaxSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ThresholdMaxLbl</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numClusters</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelCluster</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>distanceThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_distance</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NoClustersSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NoClustersLbl</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>huMomentSimilaritySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_19</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScoreThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ScoreThresholdLbl</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
